### ISSUE TITLE
Update gemspec

### ISSUE THREAD

**Issue by christinach** [pulibrary/requests#442]

**Comment by christinach** [comment]
bundle exec rake ci will fail with ruby 2.6:
```
Bundler could not find compatible versions for gem "railties":
  In snapshot (Gemfile.lock):
    railties (= 5.1.7)

  In Gemfile:
    rails (~> 5.1.7) was resolved to 5.1.7, which depends on
      railties (= 5.1.7)

    sass-rails (~> 5.0) was resolved to 5.1.0, which depends on
      railties (>= 5.2.0)

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.
```

**Comment by christinach** [comment]
I can't reproduce this error. probably it was something in my local env
